.github/workflows/nodejs.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     name: Linting
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
```

Comment:
Looks like this change breaks the commit lint job because newer versions of the checkout action only fetch one commit, not the full history. Could you change this to fetch the whole history for the lint job?

Reply:
Thanks @arthurschreiber, https://github.com/tediousjs/tedious/pull/1494/commits/76b371be4adb106b3c0b6802758a412cb90a4380 should fix this